XenModel/Messages.resx

```diff
@@ -3055,9 +3055,9 @@ Would you like to carry out the operation at this time?</value>
     <comment>Message shown when user attempts to connect to a second XE Express host from the UI</comment>
   </data>
   <data name="CONNECTION_RESTRICTED_MESSAGE_LONG" xml:space="preserve">
-    <value>Connection to server {0} restricted - a connection to a [Citrix] [XenServer product] Express Edition server ({1}) already exists.
+    <value>Connection to server {0} restricted - a connection to a [Citrix XenServer] Express Edition server ({1}) already exists.

```

Comment:
We could remove this message (and its references, including the ExpressRestriction and LicenseWarningDialog classes) because it's not applicable anymore.

Reply:
Let's keep it in feature branch. We'd better do the removal on master with a separate commit.